src/services/pipeline.py

```
@@ -149,6 +152,47 @@ def _stylizer_to_pixel_art(
     return canvas
 
 
+def pixel_art_person_controlnet(
+    image: Image.Image,
+    mask: np.ndarray,
+    stylizer: "ControlNetStylizer",
```

Ruff (0.15.5)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)